bodhi/server/templates/update.html

```diff
@@ -162,6 +162,9 @@
         // seem to break anything
         var arch = result.data.arch;
         var item = result.data.item;
+        if (item === undefined){
+            item = result.data.original_spec_nvr[0] + '#' + result.data.rev;
```

Comment:
Otherwise the code would crash here, if another result without item appears and doesn't have the required fields (`original_spec_nvr` and `rev`).

Reply:
Good call, adjusted :)